reproductions/StackExchange.Redis.StackOverflow/StackExchange.Redis.StackOverflow.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFrameworks>netcoreapp2.1</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">$(TargetFrameworks);net461</TargetFrameworks>
+    <Platforms>x64;x86</Platforms>
+    <PlatformTarget>$(Platform)</PlatformTarget>
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="StackExchange.Redis" Version="2.0.601" />
+    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.6.0-preview6.19303.8" />
```

Comment:
Do you still need this package?

Reply:
It's a transitive dependency of `StackExchange.Redis`. There's a bug in the latest release version of `System.Runtime.CompilerServices.Unsafe`, so I need to force this newer version and let automatic assembly binding redirects happen so the test can run in .NET Framework. (Works fine in .NET Core.)

See
- https://github.com/StackExchange/StackExchange.Redis/issues/1126
- https://github.com/StackExchange/StackExchange.Redis/issues/1128
- https://github.com/StackExchange/StackExchange.Redis/issues/1130#issuecomment-492106376 (and the few comments below it)